skabase/SKATestDevice/SKATestDevice.py

```diff
@@ -25,15 +23,15 @@
 
 # SKA specific imports
 from skabase import release
-from skabase.auxiliary.utils import (exception_manager, convert_api_value, coerce_value)
-from SKABaseDevice import SKABaseDevice
-
 
 file_path = os.path.dirname(os.path.abspath(__file__))
 basedevice_path = os.path.abspath(os.path.join(file_path, os.pardir)) + "/SKABaseDevice"
 sys.path.insert(0, basedevice_path)
```

Comment:
Not something to fix now, but it is pretty weird that all the devices modules have to modify the path to get imports to work. It suggests something is not quite right in how the packages are set up.

Reply:
I request you to go through the packaging and code structure and let us know your inputs on what should be the best way to package. We will be happy to improve the packaging.